### PR TITLE
Update GoC orgs for July 12

### PR DESCRIPTION
**Pull request by thriuin**
Updating the transitional organizations file for July 12th, 2018

### CODE REVIEW

File: transitional_orgs.jsonl [hunk 1]
```diff
@@ -1,4 +1,3 @@
-{"extras": [{"key": "shortform_fr", "value": "AB"}, {"key": "shortform", "value": "AB"}], "id": "436939fc-d6f6-43fc-b00e-dc134654469c", "name": "ab", "title": "Government of Alberta | Gouvernement de l'Alberta"}
```
Comment (wardi) [inline transitional_orgs.jsonl]: Should we keep this one (even if it's only on the portal and not the registry)? Or make a separate file for such orgs?

Comment (thriuin) [reply transitional_orgs.jsonl]: It's convenient to have all organizations on file if you need to quickly regenerate the site or a test site. I have no objection to two files, but with the current arrangement it is very fast to generate a new and complete orgs files with the ckanapi command.

Comment (wardi) [reply transitional_orgs.jsonl]: So, if you add this line back in when you `ckanapi load organizations` on the registry there will be a province of alberta there too. Any issues with that?

Comment (wardi) [reply transitional_orgs.jsonl]: also this file seems to be less and less transitional. If we go the two-file route we could name them `gc_orgs.jsonl` and `external_orgs.jsonl` or similar

Comment (thriuin) [reply transitional_orgs.jsonl]: That works too and it is a better naming convention

**Pull request merged**